src/api/index.tsx

```diff
@@ -67,6 +67,7 @@ export enum HttpMethod {
 
 type CallApiParams = {
   apiState: ApiState;
+  credentials?: 'omit' | 'same-origin' | 'include';
```

Comment:
I would rather see this as `includeCredentials: boolean` which would only pass `credentials: 'include'` to `fetch()` when `true`. I just don't see us ever needing the other values and I'd rather not introduce them as footguns. We would need to carefully consider the use of `omit` or `same-origin`.

Reply:
Yep, that works for me.